easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.5.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'LibTIFF'
+version = '4.5.1'
+
+homepage = 'https://libtiff.gitlab.io/libtiff/'
+description = "tiff: Library and tools for reading and writing TIFF data files"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://download.osgeo.org/%(namelower)s/']
+sources = ['tiff-%(version)s.tar.gz']
+checksums = ['d7f38b6788e4a8f5da7940c5ac9424f494d8a79eba53d555f4a507167dca5e2b']
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+dependencies = [
+    ('zlib', '1.2.13'),
+    ('libjpeg-turbo', '2.1.5.1'),
+    ('XZ', '5.4.2'),
+    ('jbigkit', '2.1'),
+    ('zstd', '1.5.5'),
+    ('libdeflate', '1.18'),
+]
+
+configopts = "--enable-ld-version-script --disable-webp --disable-sphinx "
+
+
+sanity_check_paths = {
+    'files': ['bin/tiffdump', 'bin/tiffinfo', 'include/tiff.h', 'lib/%(namelower)s.a',
+              'lib/%(namelower)s.so', 'lib/libtiffxx.a', 'lib/libtiffxx.so', 'lib/pkgconfig/%(namelower)s-4.pc'],
```

Comment:
Please use 'lib/%%(namelower)s.%s' % SHLIB_EXT instead of an explicit ".so" (or 'lib/libtiff.%s' % SHLIB_EXT) there is really no reason to use namelower here)